guides/GUIDE_FOR_LIBRARY_AUTHORS.md

```diff
@@ -104,6 +104,23 @@ Unfortunately the same behavior is not available on iOS since the behavior of na
 
 Defaults to `false`.
 
+### `scrollEdgeEffects` (>= iOS 26 only)
+
+Configures the scroll edge effect for the _content ScrollView_ (the ScrollView that is present in first descendants chain of the Screen).
+Depending on values set, it will blur the scrolling content below certain UI elements (e.g. header items, search bar) for the specifed edge of the ScrollView.
```

Comment:
```suggestion
Depending on values set, it will blur the scrolling content below certain UI elements (e.g. header items, search bar) for the specified edge of the ScrollView.
```